packages/@tinacms/cli/src/cmds/init/index.ts

```diff
@@ -127,7 +121,8 @@ export async function tinaSetup(ctx: any, next: () => void, options) {
       wrapper = true
       logger.info(
         dangerText(
-          'you will have to wrap your app in a tina-wrapper component. For more information see generated file'
+          `Heads up, to enable live-editing you'll need to wrap your page or site in Tina:\n`,
+          warnText(AppJsContent)
```

Comment:
This prints out the component for easy copy/paste, 🤔  seems ok. It only shows if they choose not to override their app.tsx

Reply:
I believe the AppJsContent is also in the getting started I frame.

Reply:
> if they choose not to override their app.tsx
Yeah this only happens when they choose not to override there App.js or App.tsx

Reply:
> I believe the AppJsContent is also in the getting started I frame.

I added it in there because it's pretty small now, they'd be able to copy and paste it quickly